peering/forms.py

```diff
@@ -655,6 +655,7 @@ class RouterForm(BootstrapMixin, forms.ModelForm):
         queryset=AutonomousSystem.objects.all(),
         query_params={"affiliated": True},
     )
+    config_context = JSONField(required=False, label="Context", widget=SmallTextarea)
```

Comment:
Maybe using `Config context` as label instead of just `Context` for better understanding.